features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/utils/custom-validation/custom-validation.js

```diff
@@ -113,4 +113,7 @@ $(document).ready(function() {
         return !regexForwardSlashAtEnd.test(value);
     }, 'Name or Context contains / at the end');
 
+    $.validator.addMethod('validateDescriptionLength', function(value, element) {
+        return value.length <= 20000;
+    }, 'maximum support 20000 charactors only');
```

Comment:
fixed in commit: https://github.com/ruks/carbon-apimgt/commit/d7e56b1d85254369d07aab9572f13d4c2120df97